src/libraries/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.TryGet.cs

```diff
@@ -76,22 +76,20 @@ public string GetComment()
         /// </exception>
         public bool GetBoolean()
         {
-            ReadOnlySpan<byte> span = HasValueSequence ? ValueSequence.ToArray() : ValueSpan;
```

Comment:
Nice find here. This was really a bug. The `span` variable was only ever used in the Debug build.